src/main/resources/templates/casInstitutionLoginView.html

```diff
@@ -51,9 +48,14 @@
                         </select>
                     </section>
 
-                    <section class="text-with-mdi" th:if=${osfCasLoginContext.institutionId}>
+                    <section class="text-with-mdi">
                         <i class="mdi mdi-checkbox-blank-off-outline mdi-before-text"></i>
-                        <span><a th:href="@{/login(campaign=institution, institutionId=${institutionId ?: ''}, service=${service?.originalUrl ?: ''})}" th:utext="#{screen.institutionlogin.link.select}"></a></span>
+                        <span th:if="${osfCasLoginContext.institutionId}">
+                            <a th:href="@{/login(campaign=institution, institutionId=${institutionId ?: ''}, service=${service?.originalUrl ?: ''})}" th:utext="#{screen.institutionlogin.link.select}"></a>
+                        </span>
+                        <span th:unless="${osfCasLoginContext.institutionId}">
+                            <a th:href="@{/login(campaign=unsupportedinstitution, institutionId=${institutionId ?: ''}, service=${service?.originalUrl ?: ''})}" th:utext="#{screen.institutionlogin.link.unsupported}"></a>
+                        </span>
```

Comment:
I combined the "I can't find my institution" and "Not your institution?" into one `if ... unless` since it always shows up no matter auto-select or not.

Reply:
Oooh, good catch!